### PR TITLE
First level of abstraction completed

### PR DESCRIPTION
**Pull request by SaurusXI**
- [X] First level - input a word to a program which outputs an image of the same word but handwritten
- [ ] Second level - change input to docx, change output to pdf. Arrange PDF

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,10 @@ This project is *work in progress*
 * Pre-requisites:
 	-  python >= 3.0
 	-  dependencies from requirements.txt
+<br>
+
+The data can be downloaded from [ here ] (https://drive.google.com/file/d/1uonG1KNWozJ41VqMxCVsnczi-HjSB3l7/view?usp=sharing)
```
Comment (L04DB4L4NC3R) [inline README.md]: Also add a password protected firefox-send link to the original dataset

File: README.md [hunk 2]
```diff
@@ -1,47 +1,30 @@
...
 </p>
 
 ---
 [![DOCS](https://img.shields.io/badge/Documentation-see%20docs-green?style=flat-square&logo=appveyor)](INSERT_LINK_FOR_DOCS_HERE) 
   [![UI ](https://img.shields.io/badge/User%20Interface-Link%20to%20UI-orange?style=flat-square&logo=appveyor)](INSERT_UI_LINK_HERE)
 
+This project is *work in progress*
 
-## Functionalities
-- [ ]  < insert functionality >
```
[the remark this answers is not in the document]
Comment (SaurusXI) [reply README.md]: Done, take a look

File: README.md (hunk 1 above)
Comment (SaurusXI) [reply README.md]: Still trying various pre-processing techniques to generate a better dataset. Will do this for the final version of the data

**Comment by Geek-ubaid** [comment]
How are you creating the hashes for respective image?

**Comment by SaurusXI** [comment]
- [X] Second level of abstraction also completed <br>
@Geek-ubaid Hashes are created using python's dictionary data-structure. Keys of hash table are image names (like 'A3.jpg') and the value is an array that describes the image. This is also described in comments in the code. <br>
Will write documentation for the app usage, and a wiki that describes what all functions do and the layout of classes.
Adding a to-do list to my repo, take a look.

**Pull request merged**